csharp/Ice/Forwarder/ForwardingServer/Forwarder.cs

```diff
@@ -23,8 +23,8 @@ public async ValueTask<OutgoingResponse> dispatchAsync(IncomingRequest request)
             target = target.ice_oneway();
         }
 
-        // Make the invocation asynchronously. If `ice_invokeAsync` throws an Ice.LocalException that cannot be
-        // marshaled (such as Ice.ConnectionRefusedException), the object adapter that calls `dispatchAsync` converts it
```

Comment:
Why remove these?
I think using quotes to distinguish between **_english_** and **_code_** is helpful to comprehension, and wish that we did more of this. Although I would use single quotes instead of backticks for these (personally).